docs/flinkDev/building.md

```diff
@@ -58,6 +58,22 @@ mvn clean install -DskipTests -Dfast
 
 The default build adds a Flink-specific JAR for Hadoop 2, to allow using Flink with HDFS and YARN.
 
+## Build PyFlink
+
+If you want to build a PyFlink package that can be used for pip installation, you need to build Flink jars first, as described in [Build Flink](##Build Flink).
+Then go to the root directory of flink source code and run this command to build the sdist package and wheel package:
+
+{% highlight bash %}
+cd flink-python; python setup.py sdist bdist_wheel
```

Comment:
`python` -> `python3` due to python27 is offer throw exception, such as: `error: invalid command 'bdist_wheel'`